src/pydocstyle/config.py

```diff
@@ -208,7 +208,10 @@ def _get_config(self, node):
         if path in self._cache:
             return self._cache[path]
 
-        config_file = self._get_config_file_in_folder(path)
+        if self._options.config_file:
```

Comment:
I have not used the parent directory lookup/merge feature, so I need someone else to make the call, but I could see a use case for pointing to a project-specific configuration file while also loading shared settings for all projects.